vita3k/kernel/src/load_self.cpp

```diff
@@ -600,6 +657,7 @@ SceUID load_self(Ptr<const void> &entry_point, KernelState &kernel, MemState &me
     const SceUID uid = kernel.get_next_uid();
     sceKernelModuleInfo->modid = uid;
     kernel.loaded_modules.emplace(uid, sceKernelModuleInfo);
+    kernel.module_uid_by_nid.emplace(module_info->module_nid, uid);
```

Comment:
This call is not thread safe (module_uid_by_nid is synchronised using export_nids_mutex and not kernel.mutex)

Reply:
Fixed. thank you.